portfolio/src/main/webapp/script.js

```diff
@@ -11,7 +11,65 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
- 
+
+
+// These variables will be used to track whenever a reply button is clicked
+// And whether the same button was clicked again in a row.
+var replyClickCounter = 0;
```

Comment:
For the counter, I find it a bit confusing because the only reason you use it seems to be check whether the click counts clicked with an odd number of times. I think it would be more straightforward to use a boolean variable like a "replyClickToogle" or something to mark if it's enabled or disabled.

Reply:
Sounds good. I'll try that later